### PR TITLE
Sentry alerts: add actionable attribute for certain FetchHearingLocationsForVeteransJob errors

### PR DESCRIPTION
**Pull request by yoomlam**
Idea to resolve noisy, non-actionable Sentry alerts, such as [Caseflow::Error::VaDotGovMissingFacilityError](https://dsva.slack.com/archives/C42P25D6H/p1629984874002400)

Found a [Slack thread](https://dsva.slack.com/archives/C3EAF3Q15/p1579129060135200) that says Bat Team can ignore these.

### Description
Add an `actionable` attribute to `Caseflow::Error::VaDotGovMissingFacilityError` Sentry alerts caused by the `FetchHearingLocationsForVeteransJob`.

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
See new RSpec tests.

### Follow-on
- [ ] If this is merged, remember to update Sentry alerts in Slack to show the `actionable` attribute.

### CODE REVIEW

File: app/jobs/fetch_hearing_locations_for_veterans_job.rb [hunk 1]
```diff
@@ -69,7 +72,8 @@ def perform
 
           break
         rescue StandardError => error
-          capture_exception(error: error, extra: { appeal_external_id: appeal.external_id })
+          actionable = NONACTIONABLE_ERRORS.include?(error.class)
```
Comment (rubaiyat22) [inline app/jobs/fetch_hearing_locations_for_veterans_job.rb:75]: should this be the following to the determine if it's actionable?
```suggestion
          actionable = !NONACTIONABLE_ERRORS.include?(error.class)
```

File: spec/jobs/fetch_hearing_locations_for_veterans_job_spec.rb [hunk 2]
```diff
@@ -254,7 +254,22 @@
...
+        end
+      end
+
+      context "when non-actionable error is thrown" do
+        let(:nonactionable_error) { Caseflow::Error::VaDotGovMissingFacilityError.new(message: "test", code: 500) }
+        before { allow_any_instance_of(GeomatchService).to receive(:geomatch).and_raise(nonactionable_error) }
+
+        it "records a geomatch error with actionable attribute = false" do
+          expect(Raven).to receive(:capture_exception).with(nonactionable_error,
+                                                            hash_including(extra: { actionable: true,
```
Comment (rubaiyat22) [inline spec/jobs/fetch_hearing_locations_for_veterans_job_spec.rb:271]: the context says 
> records a geomatch error with actionable attribute = false 

But the test is looking for actionable attribute = true
```suggestion
                                                            hash_including(extra: { actionable: false,
```

File: app/jobs/fetch_hearing_locations_for_veterans_job.rb (hunk 1 above)
Comment (yoomlam) [reply app/jobs/fetch_hearing_locations_for_veterans_job.rb:75]: Good catch!

File: spec/jobs/fetch_hearing_locations_for_veterans_job_spec.rb (hunk 2 above)
Comment (yoomlam) [reply spec/jobs/fetch_hearing_locations_for_veterans_job_spec.rb]: Well at least the context was correct. Thanks!

**Review by rubaiyat22 — APPROVED** [review]

**Pull request merged**